packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingPopper.tsx

```diff
@@ -68,10 +68,11 @@ export function TrendingPopper(props: TrendingPopperProps) {
     const position = useWindowScroll()
     useEffect(() => {
         if (!popper.current) return
-        const { top, height } = popper.current?.getBoundingClientRect()
-        if ((top < 0 && -1 * top > height) || top > document.documentElement.clientHeight)
+        const { top = Number.NaN, height = Number.NaN } = popper.current?.getBoundingClientRect() ?? {}
```

Comment:
I'm surprised that this didn't caught by the type system/

Reply:
`popper.current` has been checked above.
```ts
if (!popper.current) return
```